### ISSUE TITLE
-[__NSArrayI bk_select:]: unrecognized selector sent to instance

### ISSUE THREAD

**Issue by sryze** [zwaldowski/BlocksKit#322]
For some odd reason I'm getting the following error:

```
-[__NSArrayI bk_select:]: unrecognized selector sent to instance 0x7f7fa3874a00
2770539872
```

when using `bk_select` with an `NSArray` returned from a Core Data fetch request.

Does BlocksKit have support for arrays returned by Core Data?

**Comment by anayini** [comment]
I think I might be seeing similar behavior?  @sryze - Did you find any resolution here?

**Comment by sryze** [comment]
Unfortunately, no, I haven't found a solution for this.

**Comment by anayini** [comment]
Ah, we figured out our problem just now.  It seemed that for some reason our code was compiling, but maybe BlocksKit wasn't being linked properly into the final binary.  I actually manually added BlocksKit to the linker flags and things seem to work now.  It is probably unrelated to your NSArray returned by a Core Data Fetch request, since its an unrecognized selector.  What is the current state of your linker flags?
